deployment/live/gcp/static-ct-staging/logs/README.md

```diff
@@ -1,21 +1,21 @@
-# Arche2025h1 staging log
+# static-ct-staging log
 
 ## Overview
 
-This config uses the
-[gcp/tesseract/cloudrun](/deployment/modules/gcp/tesseract/cloudrun) module to
-run TesseraCT on Cloud Run, backed by Tessera and preloaded with
-argon2025h1 entries.
+This directory contains the config we use to deploy our staging logs. Each log
+uses
+[/deployment/modules/gcp/tesseract/gce/](/deployment/modules/gcp/tesseract/gce/)
+module to run TesseraCT on Managed Instance Groups backed by Tessera.
 
-### Update the roots
+### Update roots
 
 Run the following command from the root of the repository.
 
 It fetches roots from argon2025h1, and filters out roots that don't parse with
```

Comment:
Replace `argon2025h1` with a Google RFC 6962 production log? It's easy to forget when that's updated.